code/game/objects/items/weaponry.dm

```diff
@@ -370,11 +370,19 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	attack_verb_continuous = list("stubs", "pokes")
 	attack_verb_simple = list("stub", "poke")
 	resistance_flags = FIRE_PROOF
-	var/extended = 0
+	var/extended = FALSE
+
+/obj/item/switchblade/Initialize()
```

Comment:
it would be cool if extendable weapons were some sort of component or whatever so we dont have to copy paste tbaton shitcode every single time

Reply:
Probably would but I'm not doing that here